### PR TITLE
Feature 70 be filter

### PR DESCRIPTION
**Pull request by IgorPopach**

### CODE REVIEW

File: client/src/components/Rooms/Rooms.js [hunk 1]
```diff
@@ -43,6 +39,7 @@ class Rooms extends Component {
     axios
       .get(api, type)
       .then(res => {
+        console.log('res.data.rooms',res.data.rooms)
```
Comment (Blager89) [inline client/src/components/Rooms/Rooms.js]: so many consoles

File: client/src/components/Rooms/Rooms.js [hunk 2]
```diff
@@ -106,6 +104,7 @@ class Rooms extends Component {
   };
 
   changeDate = date => {
+    console.log('date', date)
```
Comment (Blager89) [inline client/src/components/Rooms/Rooms.js]: again

File: client/src/components/Rooms/Rooms.js [hunk 3]
```diff
@@ -115,6 +114,7 @@ class Rooms extends Component {
 
   render() {
     const { roomsDB, isLoading } = this.state;
+    console.log('roomsDB', roomsDB)
```
Comment (Blager89) [inline client/src/components/Rooms/Rooms.js]: again

File: server/routes/event/event.js [hunk 4]
```diff
@@ -19,82 +19,92 @@ const handler = {
   },
 
   async sort(ctx) {
-    let result = ctx.request.query.sort;
-    console.log('result', result)
-    let eventsArray = [...testEvents];
+    let { sort, page } = ctx.query;
+    console.log('sort', sort);
```
Comment (Blager89) [inline server/routes/event/event.js]: again

**Review by yodeco — APPROVED** [review]

**Review by YuliaFursovych — APPROVED** [review]

**Pull request merged**